com.unity.ml-agents.extensions/Runtime/Match3/AbstractBoard.cs

```diff
@@ -27,6 +27,11 @@ public abstract class AbstractBoard : MonoBehaviour
         /// </summary>
         public int NumSpecialTypes;
 
+        /// <summary>
+        /// A random seed used to generate a board, if needed.
+        /// </summary>
+        public int RandomSeed = -1;
```

Comment:
Better on the ActuatorComponent instead? That's the only thing that uses it, right?